scripts/OGC_tests/configure_TE_BASE.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Preparing TE_BASE"
+mkdir TE_BASE
+
+export TE_BASE=/home/travis/build/mozilla-sensorweb/sensorthings/TE_BASE
```

Comment:
It would be great if this could be run locally and not only in Travis. Could these paths be relative instead of absolute?